modules/core/src/lib/draw-layers.js

```diff
@@ -247,25 +252,72 @@ function drawLayersInViewport(
   logRenderStats({renderStats, pass, redrawReason, stats});
 }
 
+function getLightSources({viewport, effects}) {
+  let lightEffect;
+  if (effects && Array.isArray(effects)) {
+    for (const i in effects) {
+      const effect = effects[i];
+      if (effect instanceof LightingEffect) {
+        lightEffect = effect;
+        break;
+      }
+    }
+  }
+
+  let lightSources = {
+    ambientLight: null,
+    pointLights: [],
+    directionalLights: []
+  };
+
+  if (lightEffect && lightEffect.ambientLight) {
+    lightSources = Object.assign(lightSources, {
+      ambientLight: lightEffect.ambientLight
+    });
```

Comment:
You do not ned use `Object.assign` here, you can directly assign value to `ambientLight` as follows,
 
```
lightSources.ambientLight = lightEffect.ambientLight;
```